hy/core/language.hy

```diff
@@ -369,12 +369,15 @@
   "Chunks coll into n-tuples (pairs by default). The remainder, if any, is not
    included unless a fillvalue is specified. The step defaults to n, but can be
    more to skip elements, or less for a sliding window with overlap."
-  (setv
-   step (or step n)
-   slices (genexpr (itertools.islice coll start None step) [start (range n)]))
-  (if (is fillvalue -sentinel)
-    (apply zip slices)
-    (apply zip-longest slices {"fillvalue" fillvalue})))
+  (if (< n 1) [] (do
```

Comment:
`n` = 0 used to work, but `tee` raises an exception when it gets an argument of 0. That's why I added the check.

I agree with you that `n` < 0 should just be an error, but come to think of it, shouldn't `n` = 0 return a list of empty tuples?